country-a-service/cda-generator/src/test/java/dk/sundhedsdatastyrelsen/ncpeh/cda/EPrescriptionL1GeneratorTest.java

```diff
@@ -27,6 +29,9 @@ void generateTest() throws Exception {
     @ValueSource(strings = {"1111111118", "0201909309"})
     public void testCdaValidity(String cpr) throws Exception {
         var prescription = FmkResponseStorage.storedPrescriptions(cpr);
+        if((long) prescription.getPrescription().size() == 0){
+            return; //should this be an error? If there are no prescriptions on the person, we cannot run any significant testing
```

Comment:
Just to clarify, this addition to the code is just to NOT get an error if the test data is missing prescriptions. It is not perfect, since we want an error if no persons have prescriptions, because otherwise we cannot test it, but we should maybe freeze the testdata more than we currently do?